packages/webapp/pages/onboarding.tsx

```diff
@@ -63,22 +56,39 @@ import { ChecklistViewState } from '@dailydotdev/shared/src/lib/checklist';
 import { getPathnameWithQuery } from '@dailydotdev/shared/src/lib';
 import { webappUrl } from '@dailydotdev/shared/src/lib/constants';
 import useMutateFilters from '@dailydotdev/shared/src/hooks/useMutateFilters';
+import dynamic from 'next/dynamic';
 import { defaultOpenGraph, defaultSeo } from '../next-seo';
-import styles from '../components/layouts/Onboarding/index.module.css';
+
+const ContentTypes = dynamic(() =>
+  import(
+    /* webpackChunkName: "contentTypes" */ '@dailydotdev/shared/src/components/onboarding/ContentTypes/ContentTypes'
+  ).then((mod) => mod.ContentTypes),
+);
+const EditTag = dynamic(() =>
+  import(
+    /* webpackChunkName: "editTag" */ '@dailydotdev/shared/src/components/onboarding/EditTag'
+  ).then((mod) => mod.EditTag),
+);
+const ReadingReminder = dynamic(() =>
+  import(
+    /* webpackChunkName: "readingReminder" */ '@dailydotdev/shared/src/components/onboarding/ReadingReminder'
+  ).then((mod) => mod.ReadingReminder),
+);
+const OnboardingFooter = dynamic(() =>
+  import(
+    /* webpackChunkName: "readingReminder" */ '@dailydotdev/shared/src/components/onboarding/OnboardingFooter'
+  ).then((mod) => mod.OnboardingFooter),
+);
 
 type OnboardingVisual = {
-  showCompanies?: boolean;
```

Comment:
Decided to only support fullBackground for now, which cleans up a lot of logic:
https://dailydotdev.slack.com/archives/C02JAUF8HJL/p1729013562020949